ssdeep/ssdeep-lib/ac-fix/Makefile.am

```diff
@@ -0,0 +1,76 @@
+
```

Comment:
maybe is more legit to use patch and autogenerate the makefile, at least more portable and we will not be distributing part of his program. it can be problematic this way too..